scripts/travis-ci/before_install.bash

```diff
@@ -8,6 +8,13 @@
 MUMBLE_HOST_DEB=${MUMBLE_HOST/_/-}
 
 if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+	# Bump the apt http timeout to 120 seconds.
+	# Without this, we'd regularly get timeout errors
+	# from our MXE mirror on dl.mumble.info.
```

Comment:
*when downloading MXE from our mirror at dl.mumble.info.

it's not our mirror that causes the error

Reply:
Hmm. OK. Maybe I should've elaborated.

I think the error happens because for Fastly to serve a file, unless you've got "streaming" enabled, it'll wait serving any bytes until the file is fully downloaded to the Fastly node.

(See https://docs.fastly.com/guides/performance-tuning/improving-caching-performance-with-large-files)

Obviously, such a scenario would trigger the error messages we've been getting.

Reply:
I suppose I should mention this in the code? :)

Reply:
I've decided to leave the message as-is, and referenced my comment here from the issue.
Thanks for the review.